### PR TITLE
Staging - Include server and deployment environment

### PR DESCRIPTION
**Pull request by conquext**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,49 +1,52 @@
...
-
-While we recongise that this is an open source project, we employ you to go through our [licensing](LICENSE) for further details on how to use our codes. Also, Instrustions on how to install local version of our program is avalaible [Here.](Instruction.md)
-
-We hope you would enjoy and continue using this great service we are providing you.
-
-Thanks as you do so.
-
+# GADS 2020 Community project
+
+![Wireframe](./src/img/wireframe/wireframe.jpg)
```
Comment (Oisavictor) [inline README.md]: why is this here?

Comment (TOBECHI-GH) [reply README.md]: Trying to put Mary's wireframe in the README file

**Review by TOBECHI-GH — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Oisavictor) [reply README.md]: Okay put it at the bottom of the readme
